src/components/tableWrapper/TableWrapper.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Nice changes, however, I do see some issues with how this functions. 

1. When implementing this component, we have to actively send if we're currently on a touch screen. I think this component can figure that out on its own, making the implementation much simpler.
2. In the implementation layer, we're currently simply checking if the screen size is smaller than `x`. However, this does not necessarily mean that we're on a touch screen. Because of this, when I'm viewing a table on a small (not even that small, in my opinion) I do not see the buttons.
3. Because of point (2.) the scroll buttons don't pop up enough.
4. The scroll buttons always scroll **all the way** right or left, making the center of larger tables unreachable if you don't have a horizontal scroll function.